### PR TITLE
docs(alert): migrate docs to /elements

### LINKED ISSUE #820: [docs]: Implement new IA for rh-alert

### PR DESCRIPTION
**Pull request by bennypowers**
Closes #820

## What I did

1. 50c76b59 docs: use markdown-it-attrs
    - lets us inline styles, classes, ids, etc in markdown
1. ff62ded4 refactor(docs)!: make example shortcode an image
    - simplifies using the example docs component (todo: webc it)
1. ecc892ea docs(footer): move installation instructions to code page
3. 7cdbcdb9 docs(alert): migrate docs to /elements
4. 7170b778 docs: copy elements docs assets
5. 9f671c5b docs: unify feedback partial for footer, alert
6. 391d298d docs(footer): move assets to /elements
7. 922f16a7 docs(alert): move assets to /elements

## Testing Instructions

1. check the [dialog docs](https://deploy-preview-819--red-hat-design-system.netlify.app/components/dialog/) for visual regressions
2. check the cheatsheet for visual regressions (or skip, since it's dev-only)
3. check the footer [overview](https://deploy-preview-819--red-hat-design-system.netlify.app/components/footer/) and [code](https://deploy-preview-819--red-hat-design-system.netlify.app/components/footer/code/) tabs for sensibility
5. check the [new alert pages](https://deploy-preview-819--red-hat-design-system.netlify.app/components/alert) for regressions

DP demos:

- [alert](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/demo/) 
- [footer](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/) 
- [footer universal](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/footer-universal/)

## Notes to Reviewers

- Check for the correctness of content structure
- check the correctness of heading levels / presence of interleaving content
If enough *content* changes are likely that it will hold up merging past EoW, please make a new issue for those changes, as this PR has infra stuff in it. Alternately, we can split this into two PRs.

### CODE REVIEW

File: docs/components/dialog.md [hunk 1]
```diff
@@ -59,37 +59,38 @@ Dialogs display information in a window or help a user focus on a task without n
...
+           alt="Dialog style",
+           src="/assets/dialog/dialog-style.svg" %}
+
+{% example palette="light",
+           class="inline-flex centered",
+           style="--inline-img-max-width:872px;",
+           alt="Dialog style for video",
+           src="/assets/dialog/dialog-style-video.svg" %}
 
 #### Content
```
Comment (nikkimk) [inline docs/components/dialog.md:75]: `###` for correct heading hierarchy

File: docs/components/dialog.md [hunk 2]
```diff
@@ -246,28 +238,22 @@ Both dialog variants use [PatternFly 4 spacers](https://www.patternfly.org/v4/gu
 
 #### Desktop
```
Comment (nikkimk) [inline docs/components/dialog.md:239]: `###` for correct heading hierarchy

File: elements/rh-footer/docs/00-overview.md [hunk 3]
```diff
@@ -44,23 +42,20 @@ flexible enough to accommodate various content types and arrangements.
...
-<h5 slot="header">Status indicator</h5>
-
-The Status indicator (All systems operational) is a feature that will be added to all footers at a later date.
-
-</rh-alert>
-
-![Footer component example - Red Hat Customer Portal]({{ '/assets/footer/footer-example-cp.png' | url }})
+![Footer component example - Red Hat Customer Portal]({{ '/elements/footer/footer-example-cp.png' | url }})
 
 #### Red Hat Developer
```
Comment (nikkimk) [inline elements/rh-footer/docs/00-overview.md:52]: Should be `###` for heading hierarchy

**Comment by nikkimk** [comment]
@bennypowers 
The following are 404s:
- `https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/overview`
- `https://deploy-preview-819--red-hat-design-system.netlify.app/components/alert` (so I checked `https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/` instead)
- `https://ux.redhat.com/components/alert/demo/` (so couldn't compare to (DP demo)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/demo/])
- `https://ux.redhat.com/elements/footer/demo/` (so couldn't compare to (DP demo)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/])
- `https://ux.redhat.com/elements/footer/demo//footer-universal` (so couldn't compare to (DP demo)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/footer-universal])
- Images on the following:
   - (Alert: Styles)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/styles/]
   - (Alert: Guidelines)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/guidelines/]
   - (Footer: Styles)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/styles/]
   - (Footer: Guidelines)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/guidelines/]
   - (Footer: Accessibility)[https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/accessibility/]

File: elements/rh-alert/docs/30-code.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{% renderOverview %}{% endrenderOverview %}
```
[the remark this answers is not in the document]
Comment (bennypowers) [reply elements/rh-alert/docs/30-code.md:1]: known issue, must fix upstream in pfe-tools

**Review by nikkimk — CHANGES_REQUESTED** [review]

**Review by nikkimk — APPROVED** [review]
Let's Get This Moving! :)

**Pull request merged**